mongodb/connector/connection/connection.go

```diff
@@ -0,0 +1,250 @@
+package mongodbconnection
```

Comment:
Can't seem to comment on the .DS_Store file, can you please remove that.

Reply:
I will remove them